CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 
 * The default transformation for the calculation of the colocalization score is now `rate-diff` instead of `log1p`.
+* Rename `edge_rank_plot` function to `molecule_rank_plot`.
```

Comment:
Would it make sense to keep `edge_rank_plot` too for backwards compatibility and remove it in a future release?

Reply:
It should fail for any pixel file that is generated after 0.16.2 anyway because edge_rank_plot looks for 'edges' in adata.obs and that has been replaced 'molecules' since then. But we could keep it for older pixel files, maybe with future warning @johandahlberg?

Reply:
Yeah, I think it might be good to keep the old one around as well for now (with a deprecation notice as suggested). As this is probably a plot that people actually use.

We could also make it a bit more robust by falling back to `edges` if `molecules` is not available.

Reply:
That sounds like a reasonable solution to me!

Reply:
Now added the backward compatibility with the old function name and use of "edges" column instead of "molecules" in the new function.